.gitattributes

```diff
@@ -1,6 +1,11 @@
-/.gitignore       export-ignore
-/.gitattributes   export-ignore
-/.travis.yml      export-ignore
-/.scrutinizer.yml export-ignore
-/tests            export-ignore
-/phpunit.xml.dist export-ignore
+.gitattributes     export-ignore
+.github/           export-ignore
+.gitignore         export-ignore
+.travis.yml        export-ignore
+CHANGELOG.md       export-ignore
+CODE_OF_CONDUCT.md export-ignore
+CONTRIBUTING.md    export-ignore
+LICENSE            export-ignore
+phpunit.xml.dist   export-ignore
+README.md          export-ignore
```

Comment:
IMO, the licence, readme, changelog, and code of conduct should all remain part if the “package files.” These are files that I would want everyone to have a copy of when they install this package.

At the very least, the license file must be distributed with all downloads.